### DISCUSSION TITLE
What is the "framework" suggested for graphql

### DISCUSSION THREAD

**Discussion by baseplate-admin** [strawberry-graphql/strawberry#3497] [Q&A]
Hi, as the title suggests, i was looking for some feedback on what frameworks do you guys prioritize first?

**Comment by patrick91** [comment]
In terms of integration at the HTTP layer we don't prioritise any of them, we built a generic integration and testing suite so that every new feature works on all of them ☺️

For other integrations the one that has the most features is Strawberry Django, the integration with the ORM is 👌 (thanks to @bellini666)

We also have an integration for SQL Alchemy, but it has less features at the moment ☺️
